internal/pkg/actions/slack/slack.go

```diff
@@ -0,0 +1,62 @@
+package slack
```

Comment:
@kahootali  isn't this a copy paste of the one in Chowkidar? So, should we move into a common package and use in both?